main.tf

```diff
@@ -262,7 +264,8 @@ resource "aws_dynamodb_table" "terraform_state_lock" {
   write_capacity = 2
 
   server_side_encryption {
-    enabled = true
+    enabled     = true
+    kms_key_arn = local.dynamo_kms_master_key_id
```

Comment:
Leave out this parameter if you can for now.

Reply:
Then we need to add this in ignore file .. I added in locals like this 
` dynamo_kms_master_key_id = var.dynamo_kms_master_key_id == null ? "alias/aws/dynamo" : var.dynamo_kms_master_key_id`

Reply:
can we go ahead with this? or need to remove this parameter? @jamescrowley321

Reply:
You are good to go. I missed the local.